curriculum/challenges/english/25-front-end-development/quiz-html-tables-and-forms/66ed902df45ce3ece4053ebc.md

```diff
@@ -17,439 +17,478 @@ To pass the quiz, you must correctly answer at least 17 of the 20 of the questio
 
 ---
 
-Placeholder distractor 2
+To remove the breakline between text in a column.
 
 ---
 
-Placeholder distractor 3
+It's used to define the number of columns.
 
 #### --answer--
 
-Placeholder answer
+It merges cell across multiple columns.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+```html
+<table>
+    <tr>
+        <td colspan="3">This cell spans across three rows</td>
+    </tr>
+</table>
+```
+How to make the span across three rows only?
 
 #### --distractors--
 
-Placeholder distractor 1
+```html
+<table>
+    <tr>
+        <td row="3">This cell spans across three rows</td>
+    </tr>
+</table>
+```
 
 ---
 
-Placeholder distractor 2
+```html
+<table>
+    <tr rowspan="3">
+        <td colspan="3">This cell spans across three rows</td>
+    </tr>
+</table>
+```
 
 ---
 
-Placeholder distractor 3
+It's not possible.
 
 #### --answer--
 
-Placeholder answer
+```html
+<table>
+    <tr>
+        <td rowspan="3">This cell spans across three rows</td>
+    </tr>
+</table>
+```
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Waht is the purpose of the `thead` element?
 
 #### --distractors--
 
-Placeholder distractor 1
+It can be used interchangeably with `th`.
 
 ---
 
-Placeholder distractor 2
+It defines a single header cell.
 
 ---
 
-Placeholder distractor 3
+It is essential for marking table headers
 
 #### --answer--
 
-Placeholder answer
+It groups table header rows. 
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+The `tfoot` element can be placed before or after `tbody` element?
 
 #### --distractors--
 
-Placeholder distractor 1
+Neither before nor after
 
 ---
 
-Placeholder distractor 2
+Only Before
 
 ---
 
-Placeholder distractor 3
+Only After
 
 #### --answer--
 
-Placeholder answer
+Both before and after
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+If the `tfoot` element is written before the `tbody` element then where will it be rendered?
 
 #### --distractors--
 
-Placeholder distractor 1
+Before `tbody`.
 
 ---
 
-Placeholder distractor 2
+It will show an error.
 
 ---
 
-Placeholder distractor 3
+After that `tbody` element.
 
 #### --answer--
 
-Placeholder answer
+Always at the bottom of the table.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What element is used to define a new column in a table?
 
 #### --distractors--
 
-Placeholder distractor 1
+`th`
 
 ---
 
-Placeholder distractor 2
+`tc`
 
 ---
 
-Placeholder distractor 3
+`tcol`
 
 #### --answer--
 
-Placeholder answer
+`td`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What defines the cell of a header and improves accessibility?
 
 #### --distractors--
 
-Placeholder distractor 1
+`colspan`
 
 ---
 
-Placeholder distractor 2
+`style`
 
 ---
 
-Placeholder distractor 3
+`table` with attribute `style = " "`
 
 #### --answer--
 
-Placeholder answer
+`scope`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is `<input>` element?
```

Comment:
```suggestion
Which of the following statements is true for the `input` element?
```
`<input>` is a tag, if you talk about an element, it must be without the angular brackets.
I suggest a different question as most of the choices are not the answer to a what-question